service-model-generator/modules/service-model-generator-ls-extension/src/test/resources/get_fm_from_source/config/kafka_on_message.json

```diff
@@ -0,0 +1,248 @@
+{
+  "filePath": "sample5/main.bal",
+  "description": "",
+  "codedata": {
+    "lineRange": {
+      "fileName": "main.bal",
+      "startLine": {
+        "line": 5,
+        "offset": 4
+      },
+      "endLine": {
+        "line": 11,
+        "offset": 5
+      }
+    }
+  },
+  "response": {
+    "metadata": {
+      "label": "",
+      "description": ""
+    },
+    "kind": "OBJECT_METHOD",
+    "accessor": {
+      "metadata": {
+        "label": "Accessor",
+        "description": "The accessor of the function"
+      },
+      "valueType": "IDENTIFIER",
+      "enabled": true,
+      "editable": true,
+      "optional": false,
+      "advanced": false
+    },
+    "name": {
+      "metadata": {
+        "label": "Function Name",
+        "description": "The name of the function"
+      },
+      "valueType": "IDENTIFIER",
+      "value": "onConsumerRecord",
+      "enabled": true,
+      "editable": true,
```

Comment:
shouldn't this be `editable: false`?

Reply:
Yes. Updated in [commit](https://github.com/ballerina-platform/ballerina-language-server/pull/353/commits/1518e11bbe36a901e62819a45753821ce80a2e4a).